docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,233 @@
+# Setting Up a Cluster
+You can setup a cluster in [Kubernetes](http://kubernetes.io/docs/whatisk8s/) or Jenkins.
+
+## Setting Up a Cluster on AWS using Kubernetes
+
+### Prerequisites
+- [kubectl](http://kubernetes.io/docs/user-guide/prereqs/)
+- an [AWS](http://aws.amazon.com) account
+- [AWS CLI](https://aws.amazon.com/cli/)
+
+### Create your cluster
+A cluster consists of a master API server and a set of worker VMs called nodes.
+
+#### Using get-kube
+To start up your cluster, run:
+
+```bash
+#Using wget
+export KUBERNETES_PROVIDER=aws; wget -q -O - https://get.k8s.io | bash
+#Using cURL
+export KUBERNETES_PROVIDER=aws; curl -sS https://get.k8s.io | bash
+```
+
+This process takes about 5 to 10 minutes. User credentials and security tokens are written in `~/.kube/config`.
+By default, the script will provision a new VPC and a 4 node k8s cluster in `us-west-2a` (Oregon) with EC2 instances running on Debian.
+
+NOTE: You can override the variables defined in `config-default.sh` to change this behavior. See the [Getting Started with Kubernetes guide](http://kubernetes.io/docs/getting-started-guides/aws/) for more details.
+
+
+
+### Create a Service
+A Kubernetes [Service](http://kubernetes.io/docs/user-guide/services/) is a REST object, similar to a Pod. A Service is an abstraction which defines a logical set of Pods and a policy by which to access them - sometimes called a micro-service. The set of Pods targeted by a Service is (usually) determined by a Label Selector. We will create a Service because persist while Pods are dynamically created and destroyed regularly.
+
+To create a Service, we need to create a `service.yaml` file:
```

Comment:
Can we explain a bit about the yaml file or the service we are creating? For someone with no experience on kubernetes, it is hard to get what is going on here and what kind of service is created. >_<